### ISSUE TITLE
`x` button in accessibility help view triggers error

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode#191722]
Type: <b>Bug</b>

1. Run `Open accessibility help...`
2. Try clicking on the `x` in the help

**bug**
See the error:

```
workbench.desktop.main.js:sourcemap:765 Unable to write to User Settings because accessibility.verbosity.editor is not a registered configuration.
```

VS Code version: Code - Insiders 1.82.0-insider (Universal) (ebd67244fb2da33ab078bb2baa96106fda29f336, 2023-08-29T05:34:04.713Z)
OS version: Darwin x64 22.6.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz (16 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|3, 9, 10|
|Memory (System)|32.00GB (0.92GB free)|
|Process Argv|--crash-reporter-id 48781ca2-1705-4f64-9bab-325055aab55d|
|Screen Reader|no|
|VM|0%|
</details>

**Comment by meganrogge** [comment]
Reproduces in an untitled file because the `untitled hint` verbosity setting ID contained the `editor` verbosity setting ID cc @joyceerhl
